src/http-spec.ts

```diff
@@ -239,5 +240,5 @@ export interface IOauth2ClientCredentialsFlow extends IOauth2Flow {
 }
 
 export interface Extensions {
-  [key: string]: any;
+  [key: string]: unknown;
```

Comment:
IMHO we might consider putting this on the base node as optional. We'll ultimately want/need to support storing arbitrary properties at every level (basically a catch-all) right?

Reply:
hmmmm, I'm not so sure.
We'd probably store them in some object, wouldn't we?

Reply:
Like extensions no? extensions is an object, or maybe i'm misunderstanding

Reply:
do you mean the following?

```
extensions?: Extensions;
```

Reply:
yessir

Reply:
yeah, that's what I figured, already pushed a commit